node_agent/main.go

```diff
@@ -35,8 +35,10 @@ func init() {
 		"Node identity private key file")
 	naConfig.IstioCAAddress = flag.String("ca-address", "127.0.0.1",
 		"Istio CA address")
-	naConfig.PercentageExpirationTime = flag.Int("renew-wait-time", 60,
-		"Percentage time to elapse before renewing certificate")
+	naConfig.CertRenewalGracePeriodPercentage = flag.Int("renew-grace-period-percent",
+		na.CertRenewalGracePeriodMaxPercentage,
+		"Integer in [1, 99], grace period length in the percentage of cert TTL. For example, "+
```

Comment:
How about making the logic a little bit simpler, i.e., 30 means renewing when 30% of the TTL has passed? That would make here, and the actual code below easier (i.e., you don't need 100-na.CertRenewalGracePeriodMaxPercentage, but just use na.CertRenewalGracePeriodMaxPercentage directly)